values.yaml

```diff
@@ -1,402 +1,134 @@
-  #
-  # e.g.
-  # tolerations:
-  # - key: "my key"
-  #   operator: "Exists"
-  #   effect: "NoExecute"
-  #   tolerationSeconds: 6000
-  tolerations: []
-
-  # environments.nodeSelector -- nodeSelector is applied to all user
-  # environments to specify eligible nodes for environments to run on.
-  # See: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
-  # 
-  # eg.
-  # nodeSelector:
-  #   disktype: ssd
-  nodeSelector: {}
-
-# certs -- Describes CAs that should be added to Coder services. These certs
-# are NOT added to environments.
-certs:
-  secret:
-    # certs.secret.name -- The name of the secret.
-    name: ""
-    # certs.secret.key -- The key in the secret pointing to the certificate
-    # bundle.
-    key: ""
-
-# namespaceWhitelist -- DEPRECATED: A list of additional namespaces to which
-# environments may be deployed. Use multiple workspace providers instead.
-# Existing environments within these namespaces will continue to function, but
-# new environments cannot be created within these namespaces.
-#
-# If you used this value before 1.17, you must keep it set to the same value
-# until all environments within these namespaces are deleted, then you can set
-# it back to [].
-namespaceWhitelist: []
-
-ssh:
-  # ssh.enable -- Enables accessing environments via SSH.
-  enable: true
-
-# serviceTolerations -- Tolerations are applied to all Coder managed services.
-# Each element is a toleration object. To set user environment tolerations see
-# environments.tolerations. See values.yaml for an example.
-#
-# e.g.
-# serviceTolerations:
-# - key: "my key"
-#   operator: "Exists"
-#   effect: "NoExecute"
-#   tolerationSeconds: 6000
-serviceTolerations: []
+# Contains fields related to the Postgres backend. If providing your own
+# instance, a minimum version of Postgres 11 is required with the contrib
+# package installed.
+postgres:
+  # postgres.host -- Host of the external PostgreSQL instance.
+  host: ""
+  # postgres.port -- Port of the external PostgreSQL instance.
+  port: ""
+  # postgres.user -- User of the external PostgreSQL instance.
+  user: ""
+  # postgres.database -- Name of the database that Coder will use.
+  # You must create this database first.
+  database: ""
+  # postgres.passwordSecret -- Name of an existing secret in the
+  # current namespace with the password of the PostgreSQL instance.
+  # The password must be contained in the secret field `password`.
+  # This should be set to an empty string if the database does not
+  # require a password to connect.
+  passwordSecret: ""
+  # postgres.sslMode -- Provides variable levels of protection for
+  # the PostgreSQL connection. For acceptable values, see:
+  #  https://www.postgresql.org/docs/9.1/libpq-ssl.html
+  sslMode: "require"
 
-# Services contains global configuration that applies to all Coder managed
-# services.
+  # postgres.default -- Configure a built-in PostgreSQL deployment.
+  default:
```

Comment:
If we're gonna break this backwards compat, I think we should move to shipping the official Postgres helm as a subchart.

Reply:
Or just give a basic tutorial on how to do it in the docs with the official chart, and we ship nothing?

Reply:
This wouldn't break backwards compat as it is right now though.

Reply:
Why wouldn't it? It seems like we're moving the resources and stuff from `timescale` to `postgres.default`.

Reply:
I don't think timescale nor the postgresql images on Docker Hub support running in OpenShift, primarily because of the runAsUser uid.

CrunchyData is a company that provides a postgresql image for OpenShift, so if we do subchart stuff, that should be something that we consider.

Reply:
> I don't think timescale nor the postgresql images on Docker Hub support running in OpenShift, primarily because of the runAsUser uid.
> 
> CrunchyData is a company that provides a postgresql image for OpenShift, so if we do subchart stuff, that should be something that we consider.

I think the benefit of just using the public chart is they could easily replace the image if necessary.

Reply:
I'm in favor of getting rid of timescale in favor of postgres subchart or nothing entirely.

Reply:
@coadler we can alias them internally in the chart to make transition simple. I mean that it wouldn't create a "new" Postgres instance.

Using another chart probably isn't something I'll do here.